### ISSUE TITLE
[gtk3] [all themes] when dragging file/folder in caja, a black border is added inside pane

### ISSUE THREAD

**Issue by kerreidru** [mate-desktop/mate-themes#194]
I'm not sure if this is a theme issue or a caja issue? this started when caja switched to gtk3 ...

**Here's what caja looks like before you drag a file/folder:**
![before](https://user-images.githubusercontent.com/31544695/29976337-af09d27c-8f07-11e7-823e-98592205edf4.png)

**Here's what caja looks like once you start to drag a file/folder (notice black border added inside pane):**
![during-drag](https://user-images.githubusercontent.com/31544695/29976389-dcc03b2a-8f07-11e7-8ff1-d30166e52017.png)

this didn't occur in gtk2 version of caja.

#### MATE general version
1.18

#### Package version
mate-themes 3.22.13

#### Linux Distribution
Fedora

**Comment by raveit65** [comment]
With https://github.com/mate-desktop/caja/pull/843
we use what is setup in gtk themes for the dnd style class.
Ie. BlueMenta
https://github.com/mate-desktop/mate-themes/blob/master/desktop-themes/BlueMenta/gtk-3.0/gtk-widgets.css#L225
```
.dnd {
	border-width: 1px;
	border-style: solid;
	border-color: @theme_selected_bg_color;
	border-radius: 0;
}
```

![caja-dnd](https://user-images.githubusercontent.com/961604/29995007-dbe5e4e8-8fde-11e7-8018-9d799b25eafd.png)

**Comment by kerreidru** [comment]
looks a lot better!

**Comment by raveit65** [comment]
Btw. Do you think we need a setting in themes here?
Or is showing the drag state in caja view the **most-wanted-useless-gtk3-setting** ever? :-)

**Comment by lukefromdc** [comment]
As far as I am concerned, keeping the frame off the view is enough. Doing this for the theme as a whole and not just for Caja also fixes the same issue in Nemo and in Nautilus, where the same themes that draw the frame with this PR also draw it in Nemo, and where it is not drawn by the other themes.

Thus this PR gives behavior in Caja consistant with Nemo and Nautilus

**Comment by raveit65** [comment]
Fix will be include in next 1.18.x release,
....or maybe a bit quicker with a patch in fedora :-)
